docs/migration/index.md

```diff
@@ -0,0 +1,340 @@
+---
+order: 1
+parent:
+  title: Migration
+  order: 3
+description: Required changes when upgrading from Starport v0.17.3 to Starport v0.18.
+---
+
```

Comment:
Let's define why we need these changes, how do these changes look?

Reply:
We need these modifications because Cosmos SDK v0.44 changed a lot how things work.